File: Python/백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.py
a=int(input()); b=int(input())
if(a>=b):
    print('Congratulations, you are within the speed limit!')
else:
    if(b-a<=20):
        fine=100
    elif(b-a<=30):
        fine=270
    else:
        fine=500
    print(f'You are speeding and your fine is ${fine}.')

File: Java/백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        if(b<=a){
            System.out.println("Congratulations, you are within the speed limit!");
        }
        else{
            int cost = 0;
            if(b-a<=20){
                cost = 100;
            }
            else if(b-a<=30){
                cost = 270;
            }
            else{
                cost = 500;
            }
            System.out.printf("You are speeding and your fine is $%d.",cost);
        }
    }
}